widgets/common/src/filters/AddRuleButton.tsx

```diff
@@ -0,0 +1,15 @@
+import type { FunctionComponent } from 'react';
+import { i18nPrefix } from './consts';
+
+interface AddRuleButtonProps {
+    onClick: () => void;
```

Comment:
Optional:

One thing we can improve here is reuse the `Button`'s `onClick` type. This way add type information about the parameters that will be accessible in JavaScript (because we do not cut out the parameters that are passed to `onClick` by the button), which aligns TS and JS :slightly_smiling_face: 

```suggestion
    onClick: ComponentProps<typeof Stage.Basic.Button>['onClick'];
```

`ComponentProps` comes from `react`

Reply:
Thanks for letting me know. Applied.